app/views/layouts/_navbar.html.erb

```diff
@@ -1,5 +1,16 @@
 <nav class="navbar navbar-inverse navbar-static-top">
   <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <% if session[:organization_name] %>
```

Comment:
No this works now, its just the hamburger bar for phones (the menu bar) and the title of the organization